### DISCUSSION TITLE
how to push file to others

### DISCUSSION THREAD

**Discussion by margaretnajder** [community/community#36196] [New to GitHub]
hello 
I would like to know how i can push my file from repository to[ ](https://github.com/me50/margaretnajder.git.)
i am new here and dont know what i am doing please help

**Comment by PraaneshSelvaraj** [comment]
Hello, I think you are asking how to contribute to other repository.

## Step 1
First create a fork of that repository. This will create a copy of that repository in your account.

## Step 2
Make changes (Add your code) in that copy of that repository.

## Step 3
Once you finished your changes and commited them. You can create a Pull request.

## Step 4
Once you created a pull request. The repository maintainers will review your changes and merge it.

If you understood mark it as an answer, else ask questions.

**Comment by ossanna16** [comment]
Hey there @margaretnajder and thank you for asking a good question! I'm glad you found our community 🙂

**Comment by margaretnajder** [comment]
Thank you I still don’t understand but that ok  , I am new to this so I have to learn and read everything . Thank you

**Reply by ossanna16** [reply]
It's totally ok to learn and to ask questions. Can you please specify what you don't understand so we can try to help in a better way?

**Comment by margaretnajder** [comment]
Thank you for answer
